Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -55,8 +58,21 @@ public struct PreplannedListItemView: View {
     @ViewBuilder private var downloadButton: some View {
         switch model.status {
         case .downloaded:
-            Image(systemName: "checkmark.circle")
-                .foregroundStyle(.secondary)
+            Button {
+                Task {
+                    if let map = await model.loadMobileMapPackage() {
+                        onMapSelectionChanged?(map)
+                    }
+                }
+            } label: {
+                Text("Open")
+                    .font(.system(.headline, weight: .bold))
+                    .foregroundColor(.accentColor)
+                    .frame(width: 76, height: 32)
+                    .background(Color(UIColor.tertiarySystemFill))
+                    .cornerRadius(16)
```

Comment:
`cornerRadius` will be [deprecated](https://developer.apple.com/documentation/deviceactivity/deviceactivityreport/cornerradius(_:antialiased:)) in iOS 18. Let's use 
```suggestion
                    .clipShape(.rect(cornerRadius: 16))
```